### PR TITLE
Added cookieMaxAge in Config

### PR DESCRIPTION
**Pull request by NiketanG**
### What

Adds a custom cookieMaxAge in Medusa Config that will be used to set a custom cookie expiry time. Uses the default settings if no config value is provided.

### Why
#947

### CODE REVIEW

File: packages/medusa/src/config/index.js [hunk 1]
```diff
@@ -26,6 +26,9 @@ const config = {
   cookieSecret:
     process.env.NODE_ENV === "test" ? "test" : process.env.COOKIE_SECRET,
 
+  // Max Age for Session Cookie. 10 Hours by default
```
Comment (timtamimi) [inline packages/medusa/src/config/index.js:29]: My only nit-pick would be to consider changing the comment to use the same format as the other comments, just to keep the convention.

E.g. 
```
/**
* Max Age for Session Cookie. 10 Hours by default
*/
```

**Review by srindom — CHANGES_REQUESTED** [review]
Good idea! I think it would be nicer if this could be configured from the `medusa-config.js` file. You can use `getConfigFile` from `medusa-core-utils` to retrieve the project config :)

**Comment by NiketanG** [comment]
> Good idea! I think it would be nicer if this could be configured from the `medusa-config.js` file. You can use `getConfigFile` from `medusa-core-utils` to retrieve the project config :)

Done.

**Comment by olivermrbl** [comment]
Can I get you to change base branch to `develop` and rebase off of it? 😃 @NiketanG

**Comment by olivermrbl** [comment]
> Can I get you to change base branch to `develop` and rebase off of it? @NiketanG
> 
> _edit: It also seems, that the changes have gone missing 😄_

Any update on the missing changes here @NiketanG ? 😄

**Pull request merged**